02_Methodiek.Rmd

```diff
@@ -252,11 +252,12 @@ In dit project willen we deze analysetechniek toepassen op grondwatermeetreeksen
 
 Het eerste doel is ook al hoger ter sprake gekomen. 
 Slagen we er in om op een betrouwbare wijze de waterstandsreeksen te verlengen/vervolledigen, dan kan een meetpunt geschikt worden om in het droogtemeetnet opgenomen te worden. 
-De tijdreeksen krijgen ook een meer uniforme lengte, waardoor het afleiden van een droogte-indicator meer gestandaardiseerd wordt.
+De tijdreeksen krijgen ook een uniforme lengte, waardoor het opvolgen van de evolutie van de droogte-indicator over de jaren heen meer betekenis krijgt.
 
-Het tweede doel is in feite een bijkomende selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat de waterstanden van een meetpunt systematisch dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
-Met een systematische daling of stijging bedoelen we een lineaire trend die niet door externe factoren (opstuwen of graven greppels, waterwinning) kan verklaard worden. 
-Indien een tijdreeks een lineaire trend vertoont, wordt het meetpunt als momenteel ongeschikt beschouwd.
+Het tweede doel is in feite een bijkomende selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat, na het in rekening brengen met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijkaan dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
```

Comment:
```suggestion
Het tweede doel is in feite een bijkomend selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat, na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijk aan dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
```

Reply:
> na rekening te houden met zoveel mogelijk factoren (maar in de praktijk enkel met het weer), de waterstanden van een meetpunt geleidelijk aan dalen of stijgen

"de waterstanden van een meetpunt": in deze context beter vervangen door "de niet door het weer verklaarde component van de waterstanden van een meetpunt"?

Daarmee maak je ook duidelijker dat de _waterstanden_ an sich _wel_ een trend mogen vertonen, zolang die maar verklaard zijn door een trend in het weer. Je hebt het dus over een partiële trend (na isolatie van andere, eveneens tijdsgebonden effecten). Daarin zit volgens mij de verwarring voor de lezer, misschien kan dat nog explicieter.

Reply:
Thanks for your help to clarify this.